Cargo.toml

```diff
@@ -35,6 +35,7 @@ members = [
     "crates/solidity/outputs/spec",
     "crates/solidity/testing/perf",
     "crates/solidity/testing/sanctuary",
+    "crates/solidity/testing/sourcify",
```

Comment:
I also suggest adding an `solidity_testing_sourcify` entry in `[workspace.dependencies]` below. Under `solidity_testing_solc`.